content_sync/pipelines/definitions/concourse/e2e_test_site_pipeline.py

```diff
@@ -326,15 +304,15 @@ def __init__(self, themes_branch: str, projects_branch: str, **kwargs):
                         "OCW_IMPORT_STARTER_SLUG": settings.OCW_COURSE_STARTER_SLUG,
                         "OCW_COURSE_STARTER_SLUG": settings.OCW_COURSE_STARTER_SLUG,
                         "SENTRY_DSN": settings.OCW_HUGO_THEMES_SENTRY_DSN,
-                        "NOINDEX": course_config.values["noindex"],
+                        "NOINDEX": "true",
```

Comment:
We don't ever want test sites to be indexed. Not that this setting exactly matters here, because this is just the Playwright step. and the required env vars are just being set so that Playwright will be run with the same env settings that sites are built with.